src/prospectScraper/Program.cs

```diff
@@ -304,44 +305,35 @@ private static void CreateCombinedCSV()
             fileDest.Close();
         }
 
-        private static void CreateCombinedCSVWithExtras()
+        // Takes in CSV file name, class, and class map and returns a list of the class passed in
+        private static List<T> ListMapper<T , TMap>(string csvFileName) where TMap : ClassMap<T>
         {
-            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Status.log", "Creating the big CSV....." + Environment.NewLine);
```

Comment:
Is there a reason why you've removed the log entry?

Reply:
I think I see the issue.  The diff shows your new method at the start of the CreateCombinedCSVWithExtras method instead of at line 323, where it is on your branch, and the log entry is still there.  This was my error.